components/kubeconfig-service/go.mod

```diff
@@ -81,7 +81,7 @@ replace (
 	k8s.io/apimachinery => k8s.io/apimachinery v0.23.8
 	k8s.io/apiserver => k8s.io/apiserver v0.23.8
 	k8s.io/cli-runtime => k8s.io/cli-runtime v0.23.8
-	k8s.io/client-go => k8s.io/client-go v0.23.8
+	k8s.io/client-go => k8s.io/client-go v0.24.0-alpha.4
 	k8s.io/cloud-provider => k8s.io/cloud-provider v0.23.8
 	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.8
 	k8s.io/code-generator => k8s.io/code-generator v0.23.8
```

Comment:
All the k8s.io/.. replacements here should either be removed (if complies), or else should be bumped consistently to v0.25.0